src/editor/components/header.tsx

```diff
@@ -0,0 +1,80 @@
+import { Disclosure } from '@headlessui/react'
+import { MenuIcon, XIcon } from '@heroicons/react/outline'
+import { useRouter } from 'next/router'
+import { classNames } from '../helper/class'
+import { Logo } from './logo'
+
+const navigation = [
+  { name: 'Dashboard', href: '/' },
+  { name: 'Settings', href: '/settings' },
+  { name: 'Developer', href: '/developer' },
+]
+
+export const Header = (): JSX.Element => {
+  const router = useRouter()
+
+  return (
+    <Disclosure as="nav" className="bg-gray-800">
+      {( { open } ): any => (
+        <>
+          <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+            <div className="flex items-center justify-between h-16">
+              <div className="flex items-center">
+                <Logo />
+                <div className="hidden md:block">
+                  <div className="ml-10 flex items-baseline space-x-4">
+                    {navigation.map( ( item ) => (
+                      <a
+                        key={item.name}
+                        href={item.href}
+                        className={classNames(
+                          router.pathname === item.href
+                            ? 'bg-gray-900 text-white'
+                            : 'text-gray-300 hover:bg-gray-700 hover:text-white',
+                          'px-3 py-2 rounded-md text-sm font-medium'
+                        )}
+                      // aria-current={item.current ? 'page' : undefined}
+                      >
+                        {item.name}
+                      </a>
+                    ) )}
+                  </div>
+                </div>
+              </div>
+              <div className="-mr-2 flex md:hidden">
+                <Disclosure.Button className="bg-gray-800 inline-flex items-center justify-center p-2 rounded-md text-gray-400 hover:text-white hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-800 focus:ring-white">
+                  <span className="sr-only">Open main menu</span>
```

Comment:
Instead of writing text in the code we could use a translation file that contains all labels and such. What do you think about that?

Reply:
Good Idea, but I think this is not essential for the initial layout and could be done in a later issue 👍